### DISCUSSION TITLE
Feature Request: Individual Face Buttons

### DISCUSSION THREAD

**Discussion by garethbhughes** [rafaelvcaetano/melonDS-android#913] [Ideas]
Hey, great work on this emulator! One minor request that would hopefully be easy to implement but would make a big difference to usability: Could you please tweak the layout editor to allow the face buttons to be adjusted individually instead of as a group?

Since the shape and ergonomics of a phone are so different than a physical DS, I find that being able to use something other than the standard diamond layout for ABXY can significantly improve accuracy and playability. Menu-based RPGs and tactics games work really well with the buttons arranged around the d-pad (or even putting A in the dead zone at the center of the d-pad) to be fully playable with one thumb. I've set up several other emulators like this, but it isn't currently possible in MelonDS since the buttons are locked to their diamond layout.

Thanks!

**Comment by rafaelvcaetano** [comment]
Thanks for the feedback. Sure, I will consider this for a future release.
